app/controllers/comments_controller.rb

```diff
@@ -283,6 +283,7 @@ def unreviewed
       .unreviewed_only
       .for_display
       .page(params[:page])
+    @reviewable = (@commentable.is_a?(AdminPost) || current_user_owns?(@commentable))
```

Comment:
I think a helper method might make more sense than a variable here. We have a bunch of methods for checking which comment-related buttons to display in the comments helper -- you might even be able to reuse this one: https://github.com/otwcode/otwarchive/blob/5ea268b75c40d40b7fc1f6094bfeaacaed712257/app/helpers/comments_helper.rb#L207-L211